wiki/Performance_Troubleshooting/vi.md

```diff
@@ -0,0 +1,106 @@
+# Khắc phục sự cố hiệu năng trong osu!
+
+osu! luôn cố gắng chạy với thiết lập mặc định và sẽ cảnh báo bạn nếu bạn đang làm/chạy gì đó có thể gây ảnh hưởng tới hiệu năng của trò chơi bị giảm sút.
```

Comment:
"gây ảnh hưởng tới hiệu năng của trò chơi bị giảm sút" - feels a bit weird with grammar here. Should "bị giảm sút" be removed?

Reply:
I would prefer `gây giảm sút hiệu năng của trò chơi`
But well... shorten it sound more natural.